asset/src/job_metric_api/exporter.ts

```diff
@@ -0,0 +1,24 @@
+import promClient from 'prom-client';
+import { JobMetricAPIConfig } from './interfaces';
+
+const express = require('express');
+
+export default async function createExporter(
```

Comment:
This function should create a way to close the server during the shutdown event on the worker, since that can happen at other times.